lib/Service/PermissionService.php

```diff
@@ -270,14 +270,11 @@ public function findUsers($boardId) {
 						$this->logger->info('No circle found for acl rule ' . $acl->getId());
 						continue;
 					}
-					
-					/* $b = $circle->getMember();
-
-					foreach ($circle['members'] as $user) {
-						$a = $user;
-						//$users[$user->userId()] = new User($user);
-					}  */ 
-					
+				
+					foreach ($circle->getMembers() as $user) {
+						$iuser = $this->userManager->get($user->getUserId());
```

Comment:
Please check the result of the user manager, so we don't add users that do not exist.

Reply:
Also $user would be better fitting the general naming convention

Reply:
getUserId() is only valid if the type of the circle member is a user

Reply:
> getUserId() is only valid if the type of the circle member is a user
OK, are there any other types? Do I need to check before?

Reply:
https://github.com/nextcloud/circles/blob/00e251705362f9e20c4d2bea14020421b88c9bc3/lib/Model/BaseMember.php#L48-L51 

but deck only supports sharing with users, so that is the only one we need to handle